plugins/cpp/parser/src/clangastvisitor.h

```diff
@@ -277,6 +282,60 @@ class ClangASTVisitor : public clang::RecursiveASTVisitor<ClangASTVisitor>
     return b;
   }
 
+  bool TraverseCallExpr(clang::CallExpr* ce_)
+  {
+    _contextStatementStack.push(ce_);
+    bool b
+      = clang::RecursiveASTVisitor<ClangASTVisitor>::TraverseCallExpr(ce_);
```

Comment:
Could you please create (if it doesn't exist yet), a private `using Base = clang::RecursiveASTVisitor<ClangASTVisitor>;` in the class (this is usually the convention with Clang RASTVs), and use `Base::TraverseCallExpr(ce_);` instead? Same with every other similar construct in the code.